website/client/components/achievements/questCompleted.vue

```diff
@@ -61,6 +62,9 @@ export default {
       this.$store.dispatch('user:set', {'party.quest.completed': ''});
       this.close();
     },
+    hide (evt) {
+      if (evt.trigger === 'header-close') this.setQuestCompleted();
```

Comment:
why do we check for this specific trigger?

Reply:
We check for this specific trigger because as @Alys mentioned on  the issue, we don’t want to dismiss the modal on an ‘esc’ Key press or on clicking off the modal because they’d make it too easy to miss the modal entirely. This check differentiates the close from clicking the x vs the other two. Let me know if that doesn’t make sense or you’d like me to change it. 

I could also add a comment here for visibility because I’m assuming others will have the same question in the future.

Reply:
I guessed that was the reason, I asked because you already disabled close on esc or backdrop in the modal props so this shouldn't be necessary

Reply:
Ahh gotcha, my bad. I’ll clean it up and resubmit.